.github/workflows/e2e.yml

```diff
@@ -48,12 +48,15 @@ jobs:
       - run: yarn build
       - name: End to end tests
         id: playwright
-        uses: GabrielBB/xvfb-action@v1
-        with:
-          options: -screen 0 1280x1024x24
-          run: ${{ inputs.run-target }}
+        run: |
+          if [ $(uname) = Linux ]; then
+            /usr/bin/xvfb-run --auto-servernum -s "-screen 0 1280x1024x24" ${{ inputs.run-target }}
+          else
+            ${{ inputs.run-target }}
+          fi
```

Comment:
Maybe we can use the if syntax for github actions to test the os?


https://stackoverflow.com/a/57948488

Reply:
Thanks @jameskerr. Of those alternatives discussed at https://stackoverflow.com/a/57948488, I just now switched over to the `if [ "$RUNNER_OS" = "Linux" ]` approach and tested it out successfully in [this run](https://github.com/brimdata/zui/actions/runs/7575104954/job/20630961009), so I'll go ahead and merge with that.

I like the cleanliness of `if: matrix.os == 'ubuntu-latest'` but as discussed [here](https://stackoverflow.com/questions/60916931/github-action-does-the-if-have-an-else) the Actions `if` doesn't have an `else` and instead requires a whole 'nother `if` step with the logic reversed, which feels ugly all over again, which is why I didn't go there. 😛